core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -105,4 +110,33 @@ private function createTwoFactorPreparationListener(ContainerBuilder $container,
             ->addTag('kernel.event_listener', ['event' => 'kernel.finish_request', 'method' => 'onKernelFinishRequest'])
         ;
     }
+
+    private function createTwoFactorTrustedDevicesTokenStorage(ContainerBuilder $container, string $firewallName): void
+    {
+        $container
+            ->setDefinition('contao.security.two_factor.trusted_token_storage', new ChildDefinition('scheb_two_factor.trusted_token_storage'))
+            ->replaceArgument(2, self::TRUSTED_DEVICES_TOKEN_ID_PREFIX)
+            ->replaceArgument(3, self::TRUSTED_DEVICES_TOKEN_LIFETIME)
+        ;
+    }
+
+    private function createTwoFactorTrustedCookieResponseListener(ContainerBuilder $container, string $firewallName): void
+    {
+        $trustedCookieResponseListenerId = 'contao.listener.two_factor.trusted_cookie_response_listener.'.$firewallName;
+        $domain = '/';
+
+        if ('contao_backend' === $firewallName) {
+            $domain = '/contao';
+        }
```

Comment:
This is the path, not the domain. You sure that's the 5th argument on the service?

Reply:
Fixed in c25176a8f.